src/openforms/utils/django-two-factors.py

```diff
@@ -0,0 +1,5 @@
+from django.utils.module_loading import import_string
+
+
+def should_display_dropdown_menu(request):
+    return import_string("django_admin_index.utils.should_display_dropdown_menu")(request) and request.user.is_verified()
```

Comment:
This code is barely readable and there's no reason to do dynamic imports here

```suggestion
from django_admin_index.utils import should_display_dropdown_menu as default_should_display_dropdown_menu

def should_display_dropdown_menu(request) -> bool:
    return default_should_display_dropdown_menu(request) and request.user.is_verified()
```

Reply:
done